pkg/devices/memballoon.go

```diff
@@ -0,0 +1,47 @@
+package devices
+
+type MemballoonModel int
+
+const (
+	MemBalloonModelNone MemballoonModel = iota
+	MemBalloonModelVirtio
+	MemBalloonModelVirtioNonTransitional
+	MemBalloonModelVirtioTransitional
```

Comment:
The constants should be named `MemballoonModelSomething` instead of `MemBalloonModelSomething`.